server/utils/Result.js

```diff
@@ -0,0 +1,61 @@
+const Success = value => ({
```

Comment:
don't these need jsdoc parameters saying they are successFactories/FailFactories?

Reply:
or the the thing that the type exported is a ResultFactory?